matlabBindings/+helics/helicsBrokerSetLogFile.m

```diff
@@ -1,10 +1,10 @@
 %{
-	Set the log file on a broker.
+%	Set the log file on a broker.
 
-	@param broker The broker to set the log file for.
-	@param logFileName The name of the file to log to.
+%	@param broker The broker to set the log file for.
+%	@param logFileName The name of the file to log to.
 
-	@param[in,out] err An error object that will contain an error code and string if any error occurred during the execution of the function.
+%	@param[in,out] err An error object that will contain an error code and string if any error occurred during the execution of the function.
```

Comment:
Don't need to do this now but we might need to have some sort of script that goes through and cleans these up.  For example the err param is not used in matlab